docs/pubsub-usage.rst

```diff
@@ -45,16 +45,16 @@ List topics for the default project:
 
 .. doctest::
 
-   >>> from gcloud import pubsub
-   >>> [topic.name for topic in pubsub.list_topics()]  # API request
+   >>> from gcloud.pubsub import list_topics
```

Comment:
Consistency.  If we import other things as `from gcloud.pubsub import Foo`, then we should import the functions that way too.